### DISCUSSION TITLE
Drag and drop resources

### DISCUSSION THREAD

**Discussion by abhipsnl** [concourse/concourse#6847] [Ideas]
Can we enable a feature where I can drag and  drop resources to job? This will be great features and can solve many usa cases. 

For example I have two resources. 

Git resource 1
Git resource 2

Based on the code resource 1 linked to my job
Git 1 -> my job

So I can simply drag Git 2 resource to my job
Git 2 -> myjob 

And trigger the job,  it should execute using resource 2.

Note:
Only same resource type should be allowed. 
Git  -> git
Docker-> Docker 

Once successfully connected,  it should modify the job config in memory (and only applicable for that execution)

For other execution it should use the default conf if no changes made?

**Comment by taylorsilva** [comment]
The web UI is _mostly_ a read-only interface. We're not likely to ever work on a feature like this or accept a PR for it because it feels like the `magic` anti-pattern: https://github.com/concourse/concourse/wiki/Anti-Patterns

I could imagine a tool that let's you visualize and modify your pipeline, and then commit and set it in Concourse. That would be cool! I don't think there's a place for this in the web UI though. The current state of the code also wouldn't work well with this, especially since there's no concept for a "pipeline run" within Concourse. Concourse wouldn't know how long to keep this modified pipeline as the "true" pipeline.
